daffodil-cli/src/it/scala/org/apache/daffodil/debugger/TestCLIDebugger.scala

```diff
@@ -64,8 +64,8 @@ class TestCLIdebugger {
       shell.expect(contains("error: expression evaluation failed: Schema Definition Error:"))
       shell.expect(contains("(debug)"))
 
-      Util.expectExitCode(ExitCode.Failure, shell)
-      shell.send("quit")
+      shell.sendLine("quit")
+      Util.expectExitCode(ExitCode.BugFound, shell)
```

Comment:
No tests should ever return a BugFound exit. If that's the case, we have a bug that needs to be fixed.